python-main.js

```diff
@@ -27,8 +27,12 @@ function pythonEditor(id) {
     ACE.getSession().setTabSize(4); // Tab=4 spaces.
     ACE.getSession().setUseSoftTabs(true); // Tabs are really spaces.
     ACE.setFontSize(editor.initialFontSize);
+    $("#request-repl")[0].style.fontSize = "" + editor.initialFontSize + "px";
     editor.ACE = ACE;
 
+    // Represents the REPL terminal
+    editor.t = null;
```

Comment:
`pythonEditor` is meant to be a wrapper for ACE and represent the code editor only. 
Anything dealing with the hterm should probably go into `web_editor`.